CHANGELOG.md

```diff
@@ -19,6 +17,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 
+### Security
+
+## [3.3.1] - 2019-11-26
+
+### Added
+
+- Add support for Redis Sentinel (see `redis.failover.enable`, `redis.failover.master-name`, `redis.failover.addresses` options).
```

Comment:
The original spec in https://semver.org/spec/v2.0.0 suggests *not* adding the `Add` part.
Up until 3.3.0 it looks like we were complying with this, but now we have mixed points including `Add` and the ones not including the `And` - shall we make this consitent?

Reply:
I don't have a strong opinion on this.